### ISSUE TITLE
New-Zealand-Flooding22 images have gone missing

### ISSUE THREAD

**Issue by marklit** [opengeos/maxar-open-data#7]
The TIFFs linked to in https://raw.githubusercontent.com/opengeos/maxar-open-data/master/datasets/New-Zealand-Flooding22.geojson are offline.

This one for example returns the following XML:

https://maxar-opendata.s3.amazonaws.com/events/New-Zealand-Flooding22/ard/60/213311212301/2022-12-11/10300100DE4D9300-visual.tif

```xml
<Error>
<Code>NoSuchKey</Code>
<Message>The specified key does not exist.</Message>
<Key>events/New-Zealand-Flooding22/ard/60/213311212301/2022-12-11/10300100DE4D9300-visual.tif</Key>
<RequestId>PBMD5ZV4QDVHAC2T</RequestId>
<HostId>jMkQBeojPAnMZjzLg1LNorZgnykROPgfklr6JBWjW1HgZ2V/qoUgVXaPt+yhpZnW+bBjUBRz2BI=</HostId>
</Error>
```

**Comment by giswqs** [comment]
Maxar renamed it to Zealand-Flooding23. I forgot to remove the old one. Will remove it later today when I am in front of a computer 
https://github.com/opengeos/maxar-open-data/tree/master/datasets/New-Zealand-Flooding23

**Comment by giswqs** [comment]
Fixed in #8
